tests/roles/stop_remaining_services/tasks/main.yaml

```diff
@@ -11,32 +11,40 @@
         ["standalone.localdomain"]="{{ edpm_node_ip }}"
       )
 
+- name: Set list of compute services to stop
+  set_fact:
+    compute_services_to_stop:
+      - "tripleo_nova_compute.service"
+      - "tripleo_nova_libvirt.target"
+      - "tripleo_nova_migration_target.service"
+      - "tripleo_nova_virtlogd_wrapper.service"
+      - "tripleo_nova_virtnodedevd.service"
+      - "tripleo_nova_virtproxyd.service"
+      - "tripleo_nova_virtqemud.service"
+      - "tripleo_nova_virtsecretd.service"
+      - "tripleo_nova_virtstoraged.service"
+
+- name: Set list of Pacemaker resources to stop
+  set_fact:
+    pacemaker_resources_to_stop:
+      - "galera-bundle"
+      - "haproxy-bundle"
+      - "rabbitmq-bundle"
+
 - name: stop compute services
   no_log: "{{ use_no_log }}"
   ansible.builtin.shell: |
     {{ shell_header }}
     {{ stop_other_services_shell_vars }}
 
-    ComputeServicesToStop=(
```

Comment:
Same comment here as in the controller services stopping -- we need to execute what's written in the docs.